src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Command that handles user exiting the program
+ */
+public class ByeCommand extends Command {
+    /**
+     * determines that user wants to exit the program
```

Comment:
Shouldn't the first letter of the description be in caps?
```suggestion
     * Determines that user wants to exit the program
```